index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Huseyin Biyik's Personal Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <div class="menu-container">
+            <a class="logo-container" href="#">
+                <p>Huseyin Biyik</p>
+            </a>
+            <nav>
+                <a href="#" class="hamburger-menu">
+                    <span></span>
+                    <span></span>
+                    <span></span>
+                </a>
```

Comment:
- kindly it is important to know that you are working on the desktop version and therefore the hamburger should not appear at this time.

Reply:
Thank you for the review. But I am on the 3. day and I need to build the mobile version of it. Day 3 / Setup and mobile version skeleton.

Reply:
But you're working on the desktop version. The hamburger should not appear. Instead of Hamburger, you should have a nav that contains an <ul> and your <li>